src/main/webapp/app/assessment/assessment-instructions/assessment-instructions/assessment-instructions.component.html

```diff
@@ -25,9 +30,8 @@ <h3 *jhiExtensionPoint="overrideTitle">{{ exercise.title }}</h3>
     </ng-container>
 </jhi-expandable-section>
 
-<jhi-expandable-section headerKey="artemisApp.assessmentInstructions.assessmentInstructions">
+<jhi-expandable-section headerKey="artemisApp.assessmentInstructions.assessmentInstructions" *ngIf="showAssessmentInstructions">
     <p class="markdown-preview" [innerHTML]="gradingInstructions"></p>
```

Comment:
I’d move them to the top as well, since they are also part of the assessment and should not be hidden by other things after the assessment instructions. 
The grading instructions are comments about the general assessment

Reply:
Done in 01fa3bca2d45a4619358b8b81d7cd6aff3f22ab2